pulseapi/entries/views.py

```diff
@@ -117,6 +117,17 @@ class EntryView(RetrieveAPIView):
     pagination_class = None
 
 
+class FavoriteEntries(ListAPIView):
+
+    def get_queryset(self):
+        entries = set()
+        user = self.request.user.id
+        for f in UserFavorites.objects.filter(user=user).select_related('entry'):
```

Comment:
small nit: the interpreter doesn't care, but maintainers do, so avoiding single letter variables is generally good practice. Can we call this what it is? (like "entry" or "favorite", although with the user_favs refactor, "bookmark")